process-engine/bpmn-studio
# ⚡️ Improve Updating Solutions

## Changes

1. Add Watch Solution Functions to OpenDiagramService
2. Use Solution EventListener Instead of Polling

## Issues

Closes #1878 

PR: #1879

## How to test the changes

- Open a local solution
- Rename a diagram in the opened solution without using the BPMN Studio
- **Notice that the diagram name is updated almost immediately in the BPMN Studio..**
